internal/kafka/internal/api/dbapi/kafka_request_types.go

```diff
@@ -48,14 +49,44 @@ type KafkaRequest struct {
 	RoutesCreated bool `json:"routes_created"`
 	// Namespace is the namespace of the provisioned kafka instance.
 	// We store this in the database to ensure that old kafkas whose namespace contained "owner-<kafka-id>" information will continue to work.
-	Namespace                string `json:"namespace"`
-	ReauthenticationEnabled  bool   `json:"reauthentication_enabled"`
-	RoutesCreationId         string `json:"routes_creation_id"`
-	SizeId                   string `json:"size_id"`
-	BillingCloudAccountId    string `json:"billing_cloud_account_id"`
-	Marketplace              string `json:"marketplace"`
-	ActualKafkaBillingModel  string `json:"actual_kafka_billing_model"`
-	DesiredKafkaBillingModel string `json:"desired_kafka_billing_model"`
+	Namespace                string              `json:"namespace"`
+	ReauthenticationEnabled  bool                `json:"reauthentication_enabled"`
+	RoutesCreationId         string              `json:"routes_creation_id"`
+	SizeId                   string              `json:"size_id"`
+	BillingCloudAccountId    string              `json:"billing_cloud_account_id"`
+	Marketplace              string              `json:"marketplace"`
+	ActualKafkaBillingModel  string              `json:"actual_kafka_billing_model"`
+	DesiredKafkaBillingModel string              `json:"desired_kafka_billing_model"`
+	PromotionState           KafkaPromotionState `json:"promotion_state"`
```

Comment:
Should we use the word `State` or `Status`? this also applies to the attributes in the kafka get response

Reply:
Maybe Status would be better

Reply:
Consistency reasons seems a good argument that I already thought and I think makes sense.
I will update it to `PromotionStatus`. Also for the corresponding API attributes

Reply:
Updated